src/schoology-extractor/tests/data_generation/generation_helper.py

```diff
@@ -0,0 +1,25 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+from typing import Dict, List
+from http import HTTPStatus
+
+
+def validate_multi_status(multi_response: List[Dict]):
+    """
+    Confirm response codes for all elements of a bulk response.
+
+    Parameters
+    ----------
+    multi_response : List[Dict]
+        A list of JSON-like responses to each individual request.
+
+    Raises
+    -------
+    RuntimeError
+        If any individual operation had a non-OK response code.
+    """
+    for response in multi_response:
+        if response["response_code"] != HTTPStatus.OK:
+            raise RuntimeError(str(multi_response))
```

Comment:
Code is fine, not necessary to change. Just thought you might be interested in combining lines 23 and 24.

```python
 multi_response = [
    {"response_code":200},
    {"response_code":400},
    {"response_code":200}
]

if any(r["response_code"] != 200 for r in multi_response):
    print("yep")
```

I believe it stops processing on the first instance it finds. Of course when you throw an exception you get that same benefit...

Reply:
Nice, I'll update.